src/modules/Api.ts

```diff
@@ -262,6 +269,85 @@ export class Api {
 
     return response.data;
   }
+
+  // ============ Funding Getters ============
+
+  /**
+   * Get the current and predicted funding rates.
+   *
+   * IMPORTANT: The `current` value returned by this function is not active until it has been mined
+   * on-chain, which may not happen for some period of time after the start of the hour. To get the
+   * funding rate that is currently active on-chain, use the getMarkets() function.
+   *
+   * The `current` rate is updated each hour, on the hour. The `predicted` rate is updated each
+   * minute, on the minute, and may be null if no premiums have been calculated since the last
```

Comment:
"each minute on the minute"

Remove this since we plan to change it later?

Reply:
Not planning to change -- the premium is calculated at a random time, and the predicted rate should change at the end of the interval (on the minute)